db/data/20240507113550_set_units_to_grams.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class SetUnitsToGrams < ActiveRecord::Migration[7.1]
+  def up
+    Device.all.each do |device|
```

Comment:
As a suggestion for future, `find_each` could replace `all.each` here - it's more efficient when there are lots of records